src/stream/src/executor/mview/materialize.rs

```diff
@@ -196,7 +192,37 @@ impl<S: StateStore> MaterializeExecutor<S> {
                                 None => continue,
                             }
                         }
-                        false => {
+                        ConflictBehavior::IgnoreConflict => {
```

Comment:
please reuse the code in `ConflictBehavior::OverWrite` and `ConflictBehavior::IgnoreConflict` branch and the only difference here is just one line code.

Reply:
L147 `let mut input = self.input.execute();` takes the ownership of `self`, so it's difficult to reuse the `fill_buffer_from_chunk`  and `self.state_table.write_chunk()` in one external function.